### ISSUE TITLE
Implement Pagination Component 

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#12088]
**BLOCKED BY Remove cursor pointer for current page in pagination component, [Design system #1851](https://app.zenhub.com/workspaces/platform-design-system-5f8de67192551b0012ebb802/issues/gh/department-of-veterans-affairs/vets-design-system-documentation/1851)**

## Describe the defect
On Events, Stories, and News Releases pages, when the user hovers over the active/current page link within the pagination with a mouse, the cursor changes to a hand icon which typically indicates that the element is an actionable element. However, the current page link is not actionable, cannot be clicked on, and therefore this creates a confusing experience for visual users.

## To Reproduce
1. Go to a page on within the VAMC that has paging, like [Boston news releases](https://www.va.gov/boston-health-care/news-releases/)
2. Hover over the paging link for page 1
3. Confirm that the cursor changes to a hand
4. Click on the element
5. Confirm that nothing happens when you click the element

## Related
Design system thread identifying the issue: https://dsva.slack.com/archives/CBU0KDSB1/p1686750208280289

## AC / Expected behavior

- [x] I expect it is determined if this is using the pagination component -- sort of
- [x] I expect if the pagination component is not being used, it is implemented 
- [ ] I expect that after the design system fix is shipped, the pagination :active state does not display a hand when hovering on the page number you are currently on
- [ ] Accessibility review 

## Screenshots
![image](https://user-images.githubusercontent.com/106678594/209195612-830e96ec-385e-445a-9fb6-ea68ad9b0adb.png)

### CMS Team
Please check the team(s) that will do this work.

- [ ] `Program`
- [ ] `Platform CMS Team`
- [ ] `Sitewide Crew`
- [ ] `⭐️ Sitewide CMS`
- [ ] `⭐️ Public Websites`
- [x] `⭐️ Facilities`
- [ ] `⭐️ User support`

**Comment by davidmpickett** [comment]
Removing Lovell label as this is a problem for all VAMCs

**Comment by kmariepat-cityfriends** [comment]
after this ticket is complete: @laflannery to talk with design system team (via ticket creation) about updating the design component

**Comment by kmariepat-cityfriends** [comment]
When we get into sprint planning, consider an additional separate card to update the component itself to be accessible according to the description

**Comment by jilladams** [comment]
Was planned into Sprint 84, team ran out of time to begin. Lower priority vs. other work in Sprint 85, so not pulled in. Will get moved into an upcoming sprint.

**Comment by jilladams** [comment]
Scrum notes: 
* The behavior described only applies if you click on the pagination number for the page you're already on. In that case: hover shouldn't become a hand.
* Is using a component, not `va-pagination` bc that's a React js-y component for widgets. This is a statically built page so those buttons are just links / hrefs. 
   * The component in use here doesn't really have a name, is a CSS / HTML class `va-pagination`. Still defined by the Design System, but not a React component.
   * Design system needs to redefine the :active state here with a different cursor.
   * Ryan cut a platform support ticket, they'll fix it in next sprint.

**Comment by jilladams** [comment]
@mmiddaugh FYI that because the Design System needs to own this fix, we've redefined the ACs to cover verifying their fix when it ships. This ticket won't close in current sprint, and we'll pull it back into future sprint when their fix is ready to verify.

**Comment by jilladams** [comment]
@ryguyk can you help us find the DS github ticket for this fix? would like to link as a blocker.

**Comment by ryguyk** [comment]
Design System ticket [here](https://app.zenhub.com/workspaces/platform-design-system-5f8de67192551b0012ebb802/issues/gh/department-of-veterans-affairs/vets-design-system-documentation/1851).

**Comment by CherylEvans** [comment]
This ticket was recreated in va.gov-team repo Ticket # [64842](https://app.zenhub.com/workspaces/collaboration-cycle-reviews-61d89e9e5f4c120011a0ad37/issues/gh/department-of-veterans-affairs/va.gov-team/64842)

**Comment by mmiddaugh** [comment]
@laflannery this issue says it is blocked a design system issue: [Remove cursor pointer for current page in pagination component, Design system #1851](https://app.zenhub.com/workspaces/platform-design-system-5f8de67192551b0012ebb802/issues/gh/department-of-veterans-affairs/vets-design-system-documentation/1851) which was closed and pushed back to content build. Can you review and provide next steps when you are able?

**Comment by laflannery** [comment]
@mmiddaugh I apologize for not following up and catching this sooner. I have tracked this down and have an outstanding question on that closed ticket as to what the next step is because there are still some updates that need to occur

**Comment by laflannery** [comment]
@mmiddaugh I'm coming back to this and the original issue/request is out of date because we not have a v3 pagination component, that should be used and that takes care of the issue that was originally pointed out in the ticket description. I would like to close this ticket in favor of the more recent/relevant tickets for updating all components across all Facilities products. Let me know if that works CC: @xiongjaneg

**Comment by laflannery** [comment]
Closing based on my above comment and in the interest of keeping out tickets organized
